docs/conventions.md

```diff
@@ -25,7 +23,7 @@ Versions consist of a version name and a version code.
 
 ### Version name
 
-The following [schema](https://calver.org/) ![versioning](https://img.shields.io/badge/calver-YYYY.M.PATCH-22bfda.svg) is used for versioning.
```

Comment:
I think it is YYYY.M.PATCH, isnt it?

Reply:
Thought so as well but https://calver.org says differently:
`MM - Short month - 1, 2 ... 11, 12`
`0M - Zero-padded month - 01, 02 ... 11, 12`